### PR TITLE
Has /proc/cpuinfo generated from CPUID

### PR DESCRIPTION
**Pull request by Sonicadvance1**
Instead of having this file mostly just copy and pasted, it'll now be
generated.
We end up losing a lot of supported flags with this change, which is for
the best since we wouldn't have actually supported those things.

### CODE REVIEW

File: External/FEXCore/Source/Interface/Core/CPUID.cpp [hunk 1]
```diff
@@ -25,89 +28,321 @@ CPUIDEmu::FunctionResults CPUIDEmu::Function_0h() {
...
+    (1 <<  3) | // MWait
+    (1 <<  4) | // DS-CPL
+    (0 <<  5) | // VMX
+    (0 <<  6) | // SMX
+    (0 <<  7) | // Intel SpeedStep
+    (0 <<  8) | // Thermal Monitor 2
+    (0 <<  9) | // SSSE3
+    (0 << 10) | // L1 context ID
+    (0 << 11) | // Silicon debug
+    (1 << 12) | // FMA
```
Comment (phire) [inline External/FEXCore/Source/Interface/Core/CPUID.cpp:54]: 1. Should be labled FMA3
2. I don't think we support FMA3.

File: External/FEXCore/Source/Interface/Core/CPUID.cpp [hunk 2]
```diff
@@ -25,89 +28,321 @@ CPUIDEmu::FunctionResults CPUIDEmu::Function_0h() {
...
-      (1 << 14)   // Remove AVX512VPOPCNTDQ
-      );
+  Res.eax = 0x0;
+  Res.ebx =
+    (1 <<  0) | // FS/GS support
+    (0 <<  1) | // TSC adjust MSR
+    (0 <<  2) | // SGX
+    (0 <<  3) | // BMI1
+    (0 <<  4) | // Intel Hardware Lock Elison
+    (1 <<  5) | // AVX2 support
```
Comment (phire) [inline External/FEXCore/Source/Interface/Core/CPUID.cpp:129]: AVX2 should be disabled.

File: External/FEXCore/Source/Interface/Core/CPUID.cpp [hunk 3]
```diff
@@ -25,89 +28,321 @@ CPUIDEmu::FunctionResults CPUIDEmu::Function_0h() {
...
+  Res.eax = 0 | // Stepping
+    (0 << 4) | // Model
+    (0 << 8) | // Family ID
+    (0 << 12) | // Processor type
+    (0 << 16) | // Extended model ID
+    (0 << 20); // Extended family ID
+
+  Res.ecx =
+    (1 <<  0) | // LAHF/SAHF
+    (1 <<  1) | // 0 = Single core product, 1 = multi core product
```
Comment (phire) [inline External/FEXCore/Source/Interface/Core/CPUID.cpp:248]: More of a question than a code review: 

Should we be setting this to 0 when we set the number of threads to zero?

File: External/FEXCore/Source/Interface/Core/CPUID.cpp [hunk 4]
```diff
@@ -25,89 +28,321 @@ CPUIDEmu::FunctionResults CPUIDEmu::Function_0h() {
...
+    (1 << 22) | // MMXExt
+    (1 << 23) | // MMX
+    (1 << 24) | // FXSAVE/FXRSTOR
+    (1 << 25) | // FXSAVE/FXRSTOR Optimizations
+    (1 << 26) | // 1 gigabit pages
+    (0 << 27) | // RDTSCP
+    (0 << 28) | // Reserved
+    (1 << 29) | // Long Mode
+    (1 << 30) | // 3DNow! Extensions
+    (1 << 31);  // 3DNow!
```
Comment (phire) [inline External/FEXCore/Source/Interface/Core/CPUID.cpp:312]: Do we really want to claim to support **3DNow!**

**Review by phire — APPROVED** [review]

File: External/FEXCore/Source/Interface/Core/CPUID.cpp (hunk 1 above)
Comment (Sonicadvance1) [reply External/FEXCore/Source/Interface/Core/CPUID.cpp]: Fixed

File: External/FEXCore/Source/Interface/Core/CPUID.cpp (hunk 2 above)
Comment (Sonicadvance1) [reply External/FEXCore/Source/Interface/Core/CPUID.cpp]: Fixed

File: External/FEXCore/Source/Interface/Core/CPUID.cpp (hunk 4 above)
Comment (Sonicadvance1) [reply External/FEXCore/Source/Interface/Core/CPUID.cpp]: Disabled

File: External/FEXCore/Source/Interface/Core/CPUID.cpp (hunk 3 above)
Comment (Sonicadvance1) [reply External/FEXCore/Source/Interface/Core/CPUID.cpp:248]: Without hardware testing this on a real single core product I don't really want to fuss with it.
Apparently the last one released from Intel is a Sandy Bridge Celeron G470 CPU back in 2013.

**Pull request merged**